services/github/github-release.tester.js

```diff
@@ -38,6 +38,10 @@ t.create('Release (repo not found)')
   .get('/v/release/badges/helmets.json')
   .expectBadge({ label: 'release', message: 'no releases or repo not found' })
 
+t.create('Release (filter by prefix)')
+  .get('/v/release/expressjs/express.json?prefix=4.16')
+  .expectBadge({ label: 'release', message: 'v4.16.4' })
```

Comment:
Thank you for adding a live test, but I worry this may prove to be too brittle (and yes I realize the latest version of express at this moment is 2 minor versions ahead).

It's beneficial to have a live test here to ensure the new query param is supported, but the matching value should be the schema based `isSemver`.

I would like to include a precise test as part of this but that should be done with unit tests. Consider adding one in `github-common-release.spec.js`

Reply:
I agree that since expressjs can change their tags this test is brittle. I removed the live tests and added unit tests instead.

If you want live tests I think I have an old repo that I never plan on touching. I can add a few releases but then my username is all over your repos.